pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <groupId>com.github.cafapi</groupId>
     <artifactId>caf-common-parent</artifactId>
-    <version>3.1.0-SNAPSHOT</version>
+    <version>4.0.0</version>
```

Comment:
I didn't mean literally.  I meant the project should change to `4.0.0`, either using the Jenkins job or manually, so this string should change to `4.0.0-SNAPSHOT` and the release notes file should be appropriately renamed.